docs/vectorizer_guide.md

```diff
@@ -19,25 +19,31 @@ representing only a part of a row's data -- we've simplified the entire workflow
 
 Our system empowers you to:
 
-1. Designate any text column for embedding using customizable rules. 
-1. Automatically generate and maintain searchable embedding tables. 
-1. Keep embeddings continuously synchronized with source data (asynchronously).
-1. Utilize a convenient view that seamlessly joins base tables with their
-embeddings.
+- Designate any text column for embedding using customizable rules. 
+- Automatically generate and maintain searchable embedding tables. 
+- Keep embeddings continuously synchronized with source data (asynchronously).
+- Utilize a convenient view that seamlessly joins base tables with their embeddings.
 
-This guide offers a comprehensive overview of the Vectorizer feature,
+This page offers a comprehensive overview of Vectorizer features,
 demonstrating how it streamlines the process of working with vector embeddings
-in your database. A more detailed technical specification is available
-[here](./vectorizer.md).
+in your database. For a more detailed technical specification, see the
+[Vectorizer API reference](./vectorizer.md).
```

Comment:
Actually, just by reading `Vectorizer API reference`, I expected to find somewhere in `vectorizer.md` a compact list of all possible extension functions a user can call. 
Instead, all of them are separated into paragraphs exposing user actions, which I find it right for some use cases. However, for the use case of "I don't know what I don't know" I find it useful to have a visual of all the available functions, perhaps as a link pointing to the right paragraph where they usage are documented.

WDYT? Does it make sense? Discard if you don't see the value.

Reply:
In https://github.com/timescale/pgai/blob/vectorizer/docs/vectorizer.md, i have a list showing where to find what you need. However, it is not sorted by API name, still by action group. Are you ok with that or would you prefer by api name?